packages/commonwealth/client/scripts/helpers/feature-flags.ts

```diff
@@ -27,6 +27,7 @@ const featureFlags = {
     process.env.FLAG_KNOCK_PUSH_NOTIFICATIONS_ENABLED,
   ),
   farcasterContest: buildFlag(process.env.FLAG_FARCASTER_CONTEST),
+  newEditor: buildFlag(process.env.FLAG_NEW_EDITOR),
```

Comment:
this flag is not used anywhere?

Reply:
You're good! ha

I added it thinking I was going to use it in this branch but that was a bit too aggressive.

I'm going to use it in v2 though.  Might as well leave it.